### PR TITLE
Use the `switch` statement instead of `if`

### PR DESCRIPTION
**Pull request by rohit2sharma95**
Fix the warning about the complexity of the arrow function (by reducing the number of lines in that function) in the `build-plugins.js` file.

See warning here: https://github.com/twbs/bootstrap/runs/2770771101#step:6:41

### CODE REVIEW

File: build/build-plugins.js [hunk 1]
```diff
@@ -59,76 +59,67 @@ const defaultPluginConfig = {
...
-  ) {
-    return {
-      external: []
+  switch (pluginKey) {
+    case 'Alert':
+    case 'Offcanvas':
+    case 'Tab':
+      return defaultPluginConfig
+
+    case 'Base':
```
Comment (GeoSot) [inline build/build-plugins.js:68]: Which component is this? 😖

**Comment by GeoSot** [comment]
The re-write is ok.

However I am not sure, if this file is mirroring the latest picture of our components.

Is  there any case where we could read the "imports" during the build? It would be safer to create them dynamically

File: build/build-plugins.js (hunk 1 above)
Comment (rohit2sharma95) [reply build/build-plugins.js:68]: `BaseComponent` :thinking:
https://github.com/twbs/bootstrap/blob/b613c8dbb6c30e62adaefc669130549fd9c2e28b/build/build-plugins.js#L32

**Comment by alpadev** [comment]
@GeoSot It's possible to use functions for `external` and `globals`, so I guess it should be.

**Comment by XhmikosR** [comment]
I'd land this and then see if we can make this more robust. Also, we might need to backport any changes manually to v4-dev.

**Review by GeoSot — APPROVED** [review]

**Comment by XhmikosR** [comment]
@rohit2sharma95 can you please backport it and make a PR?

**Comment by rohit2sharma95** [comment]
@XhmikosR Seems that this PR is not that much required to be backported as there are only two checks in the v4-dev branch :thinking:

**Comment by XhmikosR** [comment]
NP, I'll remove it from backporting :)

**Pull request merged**